Add `embark` support for citation keys at point (#143)
bdarcus/bibtex-actions: README.org

So, for example, say you are working on a paper. You hold the complete super-set of items you are interested in citing at some point in that buffer.
From there, you can run different actions on the candidates at will, rather than search individually for each item you want to cite.

*** Use =bibtex-actions-at-point=
    :PROPERTIES:
    :CUSTOM_ID: use-bibtex-actions-at-point
    :END:

=M-x bibtex-actions-at-point= will run the default action on citation keys found at point directly.

> By setting =bibtex-actions-embark-dwim= to nil, other actions in =bibtex-actions-buffer-map= will be prompted if =embark= is installed. Press =RET= will run the default action.

Reviewer: ```suggestion
If =embark= is installed, setting =bibtex-actions-embark-dwim= to nil will prompt for other actions in =bibtex-actions-buffer-map= . From there, pressing =RET= will run the default action.
```
The latter change I suggest is because I was confused by `RET` in this paragraph related to the same in the previous paragraph.
